monty.h

```diff
@@ -0,0 +1,35 @@
+#ifndef
```

Comment:
Add the macro to protect double inclusion.

`#ifndef MONTY_H_`
`#define MONTY_H_`

Reply:
resolved